_documentation_/for developers/symlink-workaround.md

```diff
@@ -0,0 +1,39 @@
+## In brief
+Some tasks currently have a workaround wherein symlinks are used. This is not a problem in and of itself, but I am documenting why it is necessary as this may help other people WDLize similar programs.
+
+## Reason 1: Inputs go into different folders
+_**Important note**: This describes a Cromwell execution on a local filesystem. The details of localizing files and what folders they go into changes on different backends. Some backends may not even need this workaround._
+
+The second task of the ld-prune workflow generates chromosome-level, subset GDS files in a scattered task. The third task, which is not scattered, takes in those files as inputs in order to merge them into a single GDS file.
+
+This situation, wherein a scattered task passes in inputs to a non-scattered task, passes in each instance of the scattered task's outputs into a new folder. Let's say my scattered task runs on 5 GDS files, generating 5 subset GDS files. My subsequent task is passed in those gds files like this:
+
+<img width="623" alt="Screenshot 2021-04-09 at 3 34 00 PM" src="https://user-images.githubusercontent.com/27784612/114250466-a9331f80-9952-11eb-9e09-f114f9d89e4f.png">
+
+That is to say, each GDS file now lives in its own folder within /inputs/.
```

Comment:
The screenshot is pretty self-explanatory but should there be a mention that the folder name is made of random numbers and cannot be predicted?

Reply:
That's a good point. I added a sentence mentioning it, trying to keep it brief.